wp-content/themes/citylimits/less/typography.less

```diff
@@ -37,3 +37,137 @@ a:hover, a:active {
   }
 }
 
+.wp-block-pullquote {
+  position: relative;
+  border-color: @grey1;
+  &:before {
+    content: "\201C";
+    display: inline-block;
+    width: 1ch;
+    height: 1ch;
+    margin-left: -1ch;
+    color: @red;
+    width: 100%;
+    height: 1.5rem;
+    border-top: 1px solid;
+    border-right: 1px solid;
+    border-color: inherit;
+    position: absolute;
+    top: 0;
+    font-size: 60px;
+    margin: -10px;
```

Comment:
with `.is-style-solid-color`, this should be `-10px 10px` to position it inside the background:

Before:
![Screen Shot 2019-11-11 at 13 36 03 ](https://user-images.githubusercontent.com/1754187/68611492-451fd980-0488-11ea-9064-c7fd8d542326.png)

After:
![Screen Shot 2019-11-11 at 13 36 18 ](https://user-images.githubusercontent.com/1754187/68611497-494bf700-0488-11ea-95c7-86238eea1db4.png)